File: Assignment Programs/Assignment53/program1.java
import java.util.Scanner;

class MyArray
{
	public void ReverseArray(int arr[])
	{
		int iDigit = 0, iRev = 0, iNo = 0, temp = 0;
		for(int i=0;i<arr.length;i++)
		{
			temp = arr[i];
			iNo = arr[i];
			iRev = 0;
			while(iNo != 0)
			{
				iDigit = iNo%10;
				iRev = iRev*10 + iDigit;
				iNo = iNo/10;
			}
			arr[i] = iRev;
		}
		System.out.println("Modified array with reversed elements");
		for(int i=0;i<arr.length;i++)
		{
			System.out.println(arr[i]);
		}
	}
}

class program1
{
	public static void main(String args[])
	{
		int size = 0;
		Scanner sobj = new Scanner(System.in);
		System.out.println("Enter the size of array");
		size = sobj.nextInt();
		
		int Arr[] = new int[size];
		System.out.println("Enter the elements of array");
		for(int i=0;i<size;i++)
		{
			Arr[i] = sobj.nextInt();
		}
		MyArray mobj = new MyArray();
		mobj.ReverseArray(Arr);
	}
}

File: Assignment Programs/Assignment23/program1.c
#include<stdio.h>

void strlwrX(char *str)
{
	if(str == NULL)
	{
		return;
	}
	while(*str != '\0')
	{
		if((*str>='A') && (*str<='Z'))
		{
			*str = *str+32;;
		}
		str++;
	}
} 
int main()
{
	char arr[20];
	printf("Enter string\n");
	scanf("%[^'\n']s",arr);
	strlwrX(arr);
	printf("Modified string is %s\n",arr);
	return 0;
}